lib/ld-eventsource/impl/event_parser.rb

```diff
@@ -31,7 +32,6 @@ def initialize(lines)
       def items
         Enumerator.new do |gen|
           @lines.each do |line|
-            line.chomp!
```

Comment:
BufferedLineReader already takes care of removing the line terminator characters.